### DISCUSSION TITLE
Add description to field

### DISCUSSION THREAD

**Discussion by josejachuf** [juhaku/utoipa#432] [Q&A] [answered]
Hi
Is there any way to add a description of the fields of the structure?

Something similar to:
```
#[derive(ToSchema)]
struct Pet {
   id: u64,
   #[utoipa(desc = "this is my field description")]
   name: String,
   age: Option<i32>,
}
```

**Comment by juhaku** [comment, marked as answer]
Yes just add a doc comment for your field. Keep in mind that OpenAPI V3 does not support field descriptions on schema ref fields but only in primitive type fields.
```rust
#[derive(ToSchema)]
struct Pet {
   id: u64,
   /// This is my field description
   name: String,
   age: Option<i32>,
}
```

**Reply by josejachuf** [reply]
This works fine. 
Thanks very much!
